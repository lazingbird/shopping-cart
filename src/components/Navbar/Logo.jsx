import logo from "../../assets/muitojogo.png";
const Logo = () => {
  return <img className="display: flex h-20  p-5" src={logo} />;
};

export default Logo;
