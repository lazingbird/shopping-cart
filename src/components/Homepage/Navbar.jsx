import Logo from "./Logo";
import List from "../Helpers/List";

const Navbar = () => {
  const navItems = [
    "Steam",
    "Nintendo",
    "Playstation",
    "Xbox",
    "Cr√©ditos para Jogos",
  ];

  const userItems = ["Conta", "Carrinho"];
  return (
    <>
      <nav>
        <Logo />
        <List items={navItems} />
        <List items={userItems} />
      </nav>
    </>
  );
};

export default Navbar;
