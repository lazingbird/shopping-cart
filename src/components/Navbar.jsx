import Logo from "./Logo";
import List from "./helpers/List";

import navbarData from "../data/navbarData";

const Navbar = () => {
  return (
    <>
      <header className=" h-screen bg-purple-900">
        <nav className="display: flex h-20 items-center justify-between px-3 pt-10 text-slate-200">
          <Logo></Logo>
          <List items={navbarData.productItems} />
          <List items={navbarData.userItems} />
        </nav>
      </header>
    </>
  );
};

export default Navbar;
