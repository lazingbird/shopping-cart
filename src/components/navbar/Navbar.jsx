import Logo from "./Logo";
import List from "../helpers/List";

import navbarData from "../../data/navbarData";

const Navbar = ({ focus }) => {
  return (
    <>
      <header className=" h-screen">
        <nav className="display: flex h-20 items-center justify-between border-b-2 px-3 pb-2 pt-3 text-slate-200">
          <Logo></Logo>
          <List focus={focus} items={navbarData.productItems} />
          <List items={navbarData.userItems} />
        </nav>
      </header>
    </>
  );
};

export default Navbar;
