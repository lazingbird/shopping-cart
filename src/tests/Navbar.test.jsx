import { render, screen } from "@testing-library/react";
import Navbar from "../components/Homepage/Navbar";

describe("Navbar component", () => {
  it("renders navigation items properly", () => {
    const { container } = render(<Navbar />);
    expect(container).toMatchSnapshot();
  });
});
