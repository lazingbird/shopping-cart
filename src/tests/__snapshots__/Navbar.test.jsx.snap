// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Navbar component > renders navigation items properly 1`] = `
<div>
  <nav>
    <h1>
      MuitoJogo
    </h1>
    <ul
      class="list"
    >
      <li>
        <a
          href="#"
        >
          Steam
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          Nintendo
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          Playstation
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          Xbox
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          Cr√©ditos para Jogos
        </a>
      </li>
    </ul>
    <ul
      class="list"
    >
      <li>
        <a
          href="#"
        >
          Conta
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          Carrinho
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
